packages/overlay/src/integrations/sentry/components/traces/TraceItem.tsx

```diff
@@ -0,0 +1,98 @@
+import { Link } from "react-router-dom";
```

Comment:
I would like some feedback about this.

We had some code duplication between the trace  detail header and trace list item. They basically are the same. 

I created this component to encapsulate that but ended up having too much logic inside due to handling the 2 cases.

not sure if the best option is to create the two components (trace item, trace header) and have code duplication as both components will evolve (or I hope so 😆 )